pandas/plotting/_core.py

```diff
@@ -1494,6 +1499,7 @@ def _args_adjust(self):
 
     @classmethod
     def _plot(cls, ax, y, column_num=None, return_type='axes', **kwds):
+        from pandas.core.series import remove_na
```

Comment:
About to make a PR that makes this move and nothing else.  We'll see if Travis still complains about that.